app/web/components/Datepicker.tsx

```diff
@@ -63,10 +63,8 @@ const Datepicker = ({
           minDate={minDate}
           maxDate={maxDate}
           onChange={(date) => {
-            if (date?.isValid()) {
-              field.onChange(date);
-              onPostChange?.(date);
-            }
+            field.onChange(date);
+            onPostChange?.(date);
           }}
```

Comment:
This is the major change here, this allows a null value. We need that to set the SignupForm birthdate to null as a default. 

This also gives us more flexibility to customize validation more in parent components, i.e. SignupForm, Event forms, etc.